index.bs

```diff
@@ -864,16 +861,9 @@ enum SkippableState {"playerHandles", "adHandles", "notSkippable"};
 	element specified in the VAST document.
 		- {{CreativeData/adParameters}} ad parameters from VAST, or an empty string
 			if unknown.
-		- {{CreativeData/adId}} the ID of the ad from VAST, or an empty string if
-			unknown.
-		- {{CreativeData/creativeId}} the ID from the creative or an empty string
-			if unknown.
-		- {{CreativeData/adServingId}} Quasi-unique id generated by ad server and
-			passed through all 1st and 3rd party reporting to facilitate the marriage
-			of impression-level data across multiple reporting systems. In VAST 4.1
-			and later this id is provided in the `AdServingID` node.
 		- {{CreativeData/clickThruUrl}} The click through url, provided from VAST.
-			If there is no click through url this should be an empty string.
+			If there is no click through url this should be an empty string 
```

Comment:
Why do we want multiple possibilities here?

Reply:
I updated the PR (I think). Please check.